### PR TITLE
Update example code to use SDK 2.1.0

### PR DESCRIPTION
**Pull request by frojasg**

### CODE REVIEW

File: connect-examples/v2.1/rails_payment/README.rdoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* bundle install
+
+* bundle exec rake db:create db:migrate # (No db in example, but keeps rails from complaining)
+
+* Add a .env file at the root with following values:
+
+    SQUARE_APPLICATION_ID=your-app-id
+    SQUARE_ACCESS_TOKEN=your-access-token
+
+* `bundle exec foreman start`
```
Comment (frojasg) [inline connect-examples/v2.1/rails_payment/README.rdoc]: I included foreman to simplify the instructions

File: connect-examples/v2.1/rails_payment/app/controllers/charges_controller.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        :currency => 'USD'
+      },
+      :idempotency_key => SecureRandom.uuid
+    }
+
+    # The SDK throws an exception if a Connect endpoint responds with anything besides 200 (success).
+    # This block catches any exceptions that occur from the request.
+    locations_api = SquareConnect::LocationsApi.new
+    begin
+      locations = locations_api.list_locations.locations
```
Comment (frojasg) [inline connect-examples/v2.1/rails_payment/app/controllers/charges_controller.rb]: I'm getting the locations from API call instead of hardcoding the id

**Comment by frojasg** [comment]
@yonpols just to be clear. do you mean the `php_checkout` code or the `php_payments` example? or both?

**Comment by yonpols** [comment]
I mean php checkout examples

**Comment by tristansokol** [comment]
The v2, v1 folders are representative of our API version, not our SDK versions. We should remove the  version folder structure and instead organize the samples by product (transactions, items, etc).

**Pull request merged**